src/capi.rs

```diff
@@ -738,6 +797,81 @@ pub unsafe extern fn rav1e_twopass_out(ctx: *mut Context) -> *mut Data {
   }))
 }
 
+/// Rate Control Data
+pub enum RcDataKind {
+  /// A Rate Control Summary Packet
+  ///
+  /// It is emitted once, after the encoder is flushed.
+  ///
+  /// It contains a summary of the rate control information for the
+  /// encoding process that just terminated.
+  Summary,
+  /// A Rate Control Frame-specific Packet
+  ///
+  /// It is emitted every time a frame is processed.
+  ///
+  /// The information contained is required to encode its matching
+  /// frame in a second pass encoding.
+  Frame,
+}
+
+/// Return the first pass data
+///
+/// Call it after rav1e_receive_packet().
+///
+/// use rav1e_data_unref() to free the data.
+///
+/// It will return a `RcDataKind::Summary` once the encoder is flushed.
```

Comment:
Are we expected to write this `Summary` data at the end, since no function was added for getting the size of the summary data? Or do we use the old/existing one?

I also think the old ratecontrol API should be removed while this one is added, as discussed on the watercooler. *Especially* if they share functions, since that is very confusing.

Reply:
I will remove them. 

Every time you call this function you get a packet of data. Once the encoder if flushed you receive the Summary and you can write it back up or in a separate file or at the end as you like.

Reply:
Removed.